src/app/functions/universal/scanResult.ts

```diff
@@ -4,6 +4,7 @@
 
 // TODO: Move pure functions that operate on scan results to this file
 
+// TODO: removal_under_maintenance does not belong here
```

Comment:
Question: Should we remove it then or at least link an issue for us to take care of it?

Reply:
Yea it should be removed

Reply:
The status is present in the other file `dataBroker.ts`